include/eve/module/core/regular/impl/simd/x86/is_less.hpp

```diff
@@ -19,58 +19,151 @@
+        else if constexpr (c == category::int32x8)    return mask8 {_mm256_cmplt_epi32_mask(a, b)};
+        else if constexpr (c == category::int32x4)    return mask8 {_mm_cmplt_epi32_mask(a, b)};
+        else if constexpr (c == category::int16x32)   return mask32 {_mm512_cmplt_epi16_mask(a, b)};
+        else if constexpr (c == category::int16x16)   return mask16 {_mm256_cmplt_epi16_mask(a, b)};
+        else if constexpr (c == category::int16x8)    return mask8 {_mm_cmplt_epi16_mask(a, b)};
+        else if constexpr (c == category::int8x64)    return mask64 {_mm512_cmplt_epi8_mask(a, b)};
+        else if constexpr (c == category::int8x32)    return mask32 {_mm256_cmplt_epi8_mask(a, b)};
+        else if constexpr (c == category::int8x16)    return mask16 {_mm_cmplt_epi8_mask(a, b)};
+      }
+      else
+      {
+        if      constexpr (c == category::float32x8) return _mm256_cmp_ps(a, b, f);
+        else if constexpr (c == category::float64x4) return _mm256_cmp_pd(a, b, f);
+        else if constexpr (c == category::float32x4) return _mm_cmplt_ps(a, b);
+        else if constexpr (c == category::float64x2) return _mm_cmplt_pd(a, b);
+        else
+        {
+          constexpr auto use_avx2   = current_api >= avx2;
+          constexpr auto use_sse4_1 = current_api >= sse4_1;
+          constexpr auto lt         = []<typename E>(E ev, E fv) { return as_logical_t<E>(ev < fv); };
+
+          [[maybe_unused]] auto unsigned_cmp = [](auto lhs, auto rhs)
+          {
+            using l_t     = logical<wide<T, N>>;
+            auto const sm = signmask(as<as_integer_t<wide<T, N>, signed>>{});
+            return bit_cast((bit_cast(lhs, as(sm)) - sm) < (bit_cast(rhs, as(sm)) - sm), as<l_t>{});
+          };
+
+          if      constexpr (use_avx2 && c == category::int64x4)   return _mm256_cmpgt_epi64(b, a);
+          else if constexpr (use_avx2 && c == category::uint64x4)  return unsigned_cmp(a, b);
+          else if constexpr (use_avx2 && c == category::int32x8)   return _mm256_cmpgt_epi32(b, a);
+          else if constexpr (use_avx2 && c == category::uint32x8)  return eve::min(a, b) != b;
+          else if constexpr (use_avx2 && c == category::int16x16)  return _mm256_cmpgt_epi16(b, a);
+          else if constexpr (use_avx2 && c == category::uint16x16) return eve::min(a, b) != b;
+          else if constexpr (use_avx2 && c == category::int8x32)   return _mm256_cmpgt_epi8(b, a);
+          else if constexpr (use_avx2 && c == category::uint8x32)  return eve::min(a, b) != b;
+          else if constexpr (c == category::int32x4)               return _mm_cmplt_epi32(a, b);
+          else if constexpr (c == category::int16x8)               return _mm_cmplt_epi16(a, b);
+          else if constexpr (c == category::int8x16)               return _mm_cmplt_epi8(a, b);
+          else if constexpr (c == category::uint32x4)
+          {
+            if constexpr (use_sse4_1) return eve::min(a, b) != b;
```

Comment:
I am surprised you didn't need to forward anything. I'm pretty sure max/min rely on less.
Can you check how that works?